e2e-test/cypress/plugins/index.js

```diff
@@ -150,7 +150,11 @@ module.exports = (on, config) => {
     readExcelSheet: readExcelSheet,
     deleteFile: deleteFile,
     checkFileExists: checkFileExists,
-    modifyHash: modifyHash
+    modifyHash: modifyHash,
+    log(message) {
+      console.log(message);
+      return null;
+    }
```

Comment:
if you run the e2e-test via command line (npm run e2e-tests) you couldn't log to the terminal.
This change enables logging via `cy.task('log', 'This will be output to the terminal')`